<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicJobProgressWidget</class>
 <widget class="QWidget" name="BasicJobProgressWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>95</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="job_title">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string comment="KDE::DoNotExtract">TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="job_description">
     <property name="text">
      <string comment="KDE::DoNotExtract">TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QProgressBar" name="progress">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="job_speed">
       <property name="text">
        <string comment="KDE::DoNotExtract">TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="msg">
     <property name="text">
      <string comment="KDE::DoNotExtract">TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
